src/module/actor/entity.js

```diff
@@ -33,6 +33,15 @@ export class OseActor extends Actor {
     return super.createEmbeddedDocuments(embeddedName, data, context);
   }
 
+  skipRollDialogCheck(options) {
```

Comment:
I think this ought to not be public. We could...
* mark this as private (with a pound sign in front of the name, like we're doing in the data models)
* mark it as static (since it's not using anything from the class)
* move it out of the class and put it in a collection of helper methods.

Thoughts?

Reply:
I tried marking it as private, but couldn't get it to work (my knowledge is very limited on this), I opted for the last option with my latest commit. (Ps. thanks a lot for your patience!)

Reply:
Thank *you* for your patience on all of my comments, hahaha.